src/main/java/cn/nukkit/item/Item.java

```diff
@@ -462,6 +470,26 @@ public static Item fromJson(Map<String, Object> data) {
         return items;
     }
 
+    public static CompoundTag parseCompoundTag(byte[] tag) {
+        try {
+            return NBTIO.read(tag, ByteOrder.LITTLE_ENDIAN);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
```

Comment:
what exactly way the point of moving this method?

Reply:
Auto Code Cleanup on my IDE